README.md

```diff
@@ -228,7 +228,7 @@ Uniswap for a profit.
 To run the bot, you'll need the address of a deployed version of the Liquidator
 contract (or you can deploy a new instance of it yourself):
 
-`LIQUIDATOR_ADDRESS="0xABC..." yarn liquidation-bot`
+`LIQUIDATOR_ADDRESS="0xABC..." MARKET="01-USDC" yarn liquidation-bot --network kovan`
```

Comment:
As far as I can tell, the only way to pass arguments to Hardhat scripts (anything run via `hardhat run ...`) is to use environment variables.

https://hardhat.org/hardhat-runner/docs/advanced/scripts#hardhat-arguments

It's definitely clunky to have to pass in some arguments via env variables and some (like `--network`) via params